framework/src/controller/controller.js

```diff
@@ -307,7 +307,7 @@ class Controller {
 		this.childrenList.forEach(child => child.kill());
 
 		try {
-			await this.bus.cleanup();
+			this.bus.cleanup();
```

Comment:
See the inconsistence pattern here. One place you have `async` cleanup other place you don't. We kept it consistently `async` for this purpose.